### ISSUE TITLE
twisted.words.im.tap / proxyui is untested and riddled with bugs.

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#1823]
|[<img alt="jerub's avatar" src="https://avatars.githubusercontent.com/u/315461?s=50" width="50" height="50">](https://github.com/jerub)| @jerub reported|
|-|-|
|Trac ID|trac#1823|
|Type|enhancement|
|Created|2006-06-14 22:58:28Z|
|Branch|https://github.com/twisted/twisted/tree/remove-tap-proxyui-1823|

This code should either be properly tested or removed.

<details><summary>Searchable metadata</summary>

```
trac-id__1823 1823
type__enhancement enhancement
reporter__jerub jerub
priority__normal normal
milestone__ 
branch__branches_remove_tap_proxyui_1823 branches/remove-tap-proxyui-1823
branch_author__jesstess jesstess
status__closed closed
resolution__fixed fixed
component__words words
keywords__ 
time__1150325908000000 1150325908000000
changetime__1277601724000000 1277601724000000
version__None None
owner__ 
cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_1"></a>|@jesstess commented|
|-|-|

(In [29293]) Branching to 'remove-tap-proxyui-1823'

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_2"></a>|@jesstess removed owner|
|-|-|

Remove twisted.words.im.proxyui and twisted.words.im.tap.

They are not used anywhere, do not have tests, and have not been updated in 6 years.

**Comment by twisted-trac** [comment]
|[<img alt="jerub's avatar" src="https://avatars.githubusercontent.com/u/315461?s=50" width="50" height="50">](https://github.com/jerub)<a name="note_3"></a>|@jerub commented|
|-|-|

Jess,

Thanks for looking at this ancient bug and getting it resolved. Please merge the change.

**Comment by twisted-trac** [comment]
|[<img alt="jerub's avatar" src="https://avatars.githubusercontent.com/u/315461?s=50" width="50" height="50">](https://github.com/jerub)|@jerub set owner to @jesstess|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_5"></a>|@jesstess set status to `closed`|
|-|-|

(In [29392]) Merge remove-tap-proxyui-1823

Author: jesstess
Reviewer: jerub
Fixes: [#1823](https://github.com/twisted/twisted/issues/1823)

Remove twisted.words.im.proxyui and twisted.words.im.tap.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
